diff -ur aegis-3.18.orig/common/config.h.in aegis-3.18/common/config.h.in
--- aegis-3.18.orig/common/config.h.in	Fri Jul  9 00:36:21 1999
+++ aegis-3.18/common/config.h.in	Sat Jul 17 00:00:44 1999
@@ -290,8 +290,8 @@
  * This is mostly to ensure that accounts "root" and "bin"
  * and "uucp" are not project owners.
  */
-#define AEGIS_MIN_UID 100
-#define AEGIS_MIN_GID 10
+#define AEGIS_MIN_UID 500
+#define AEGIS_MIN_GID 500
 
 /*
  * There is more to do, but we need to insulate it from config.status,
diff -ur aegis-3.18.orig/configure aegis-3.18/configure
--- aegis-3.18.orig/configure	Fri Jul  9 00:36:21 1999
+++ aegis-3.18/configure	Sat Jul 17 00:00:25 1999
@@ -2531,8 +2531,8 @@
 
 echo $ac_n "checking for owner of aegis library files""... $ac_c" 1>&6
 echo "configure:2534: checking for owner of aegis library files" >&5
-AEGIS_UID=3
-AEGIS_GID=3
+AEGIS_UID=65
+AEGIS_GID=65
 if test -f ${sharedstatedir}/lockfile -o -f ${libdir}/lockfile ; then
 	cat > conftest.$ac_ext << 'EOF'
 #line 2539 "configure"
